packages/cli-tool/src/hooks/initialize/generate-base-app.ts

```diff
@@ -0,0 +1,14 @@
+import runCommand from '../../helpers/child-process'
+
+const hook = async function(options: {
```

Comment:
Should we change this to arrow function as well?

Reply:
Yes, thanks. Updated in [5b4245b](https://github.com/nimblehq/react-templates/pull/95/commits/5b4245b17ab1cf5f471afce2030f0c9d1a72fe4b)